.travis.yml

```diff
@@ -51,6 +51,8 @@ before_script:
 script:
   # Run tests, including PEP8 check, and produce a coverage report
   - py.test --doctest-modules --cov --cov-report term-missing --pep8 $TRAVIS_BUILD_DIR/odl
+  # Invoke also the alternative way of running the unit tests
```

Comment:
I mean, it doesn't have to install everything again. It's just running the tests again, which is somewhat negligible to the 2 1/2 minutes (?) of the overall procedure.